tests/test_license.py

```diff
@@ -108,6 +108,8 @@ def setUp(self):
             'src/opendr/perception/facial_expression_recognition/landmark_based_facial_expression_recognition',
             'projects/opendr_ws_2/src/opendr_perception/test',
             'projects/opendr_ws_2/src/opendr_ros2_bridge/test',
+            'projects/opendr_ws_2/src/vision_opencv',
+            'projects/opendr_ws_2/install',
```

Comment:
It makes it impossible to pass all tests while having ros2 workspace configured, does not look good for both development and user experimentation

Reply:
You're right about that, it stuck out to me because i didn't find any other - let's say - "local" stuff being excluded here. Do you think we should add some stuff about ROS1 workspace too or it is not needed?

Reply:
Ok, i'll look into it!